static/defaults.config.php

```diff
@@ -122,6 +122,10 @@
 		// Display "Emoji Only" posts in big.
 		'big_emojis' => false,
 
+		// bulk_delivery (Boolean)
+		// Delivers AP messages in a bulk (experimental)
```

Comment:
Please explain the trade-off of this setting, does it improve performance? At what cost?

Reply:
It is completely experimental and I **hope** that it will improve performance. And when it works then it could replace the old mechanism completely. But currently I do have some issues. The `experimental` should say "please don't use at the moment".